static/docs/user-guide/external-data/http.md

```diff
@@ -0,0 +1,66 @@
+# External Data on HTTP/HTTPS
+
+**❗ Note:** HTTP/HTTPS remotes support only download operations.
```

Comment:
it means probably that we support only dependencies, right? use the same terminology ... not download operations

Reply:
> it means probably that we support only dependencies, right? use the same terminology ... not download operations

I think it is better to remove this note, since it says the obvious (there is only the section for dependencies, no section for outputs).

Reply:
We've seen people asking us over and over again - thus this note. It was not cleat either docs are out of date (always) or what :)

Reply:
Maybe it was useful when HTTP was in the same page with the other remotes, but in this page it does not make sense.
Probably people are confused by Git remotes, which can be used both to pull and push.